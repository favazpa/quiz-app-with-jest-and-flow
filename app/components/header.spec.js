// @flow
import React from 'react';
import {Component} from 'react';
import Header from './header';
import ShallowRenderer from 'react-test-renderer/shallow';

let renderer = new ShallowRenderer();

describe('src/component/header', () => {
  const renderButton = () => {
    renderer.render(<Header />);
    return renderer.getRenderOutput();
  };

  test('should render correctly', () => {
    const result = renderButton();
    expect(result).toMatchSnapshot();
  });
});