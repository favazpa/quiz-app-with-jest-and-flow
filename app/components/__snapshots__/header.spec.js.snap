// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/component/header should render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#4d6680",
      "borderBottomLeftRadius": 15,
      "borderBottomRightRadius": 15,
      "height": 70,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Text
    style={
      {
        "color": "orange",
        "fontSize": 24,
        "fontWeight": "900",
      }
    }
  >
    General Knowledge
  </Text>
</View>
`;
